modules/index.js

```diff
@@ -0,0 +1,94 @@
+import UI from './User.js';
+import Storage from './storage.js';
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+// Book class
+class Book {
+  constructor(title, author, index) {
+    this.title = title;
+    this.author = author;
+    this.index = index;
+  }
```

Comment:
- [ ] Great job working on this milestone. As per project requirements in setting up the project structure, index.js is not  to be inside the folder modules. Kindly re-arrange your project structure
![Screen Shot 2023-02-21 at 3 28 49 AM](https://user-images.githubusercontent.com/62849872/220333259-6a688133-08b6-4749-8627-2ab994bbec72.png)